src/main/kotlin/no/nav/familie/ef/sak/iverksett/IverksettingDtoMapper.kt

```diff
@@ -204,13 +204,15 @@ fun Vilkårsvurdering.tilIverksettDto(): VilkårsvurderingDto = Vilkårsvurderin
         }
 )
 
-fun PeriodeWrapper.tilIverksettDto(): List<VedtaksperiodeDto> = this.perioder.map {
-    VedtaksperiodeDto(fraOgMed = it.datoFra,
-                      tilOgMed = it.datoTil,
-                      aktivitet = AktivitetType.valueOf(it.aktivitet.name),
-                      periodeType = VedtaksperiodeType.valueOf(it.periodeType.name)
-    )
-}
+fun PeriodeWrapper.tilIverksettDto(): List<VedtaksperiodeDto> = this.perioder
+        .filter { it.periodeType != no.nav.familie.ef.sak.vedtak.domain.VedtaksperiodeType.MIDLERTIDIG_OPPHØR }
```

Comment:
Skal vi ikke returnere opphørsperioder til frontend når man går in på en tdligere behandling? Eller hva gjør denne filtrering?

Reply:
Dette er periodene som skal sendes til iverksetting - der tror jeg ikke vi skal ha med opphørsperiodene?

Reply:
Ah såklart, fint 👍